python/vision_explanation_methods/evaluation/pointing_game.py

```diff
@@ -140,6 +146,9 @@ def calculate_gt_salient_pixel_overlap(self,
         """
         Calculate percent of overlap between salient pixels and gt bbox.
 
+        Formula: number of salient pixels in the gt bbox /
```

Comment:
nit: maybe specify gt = ground truth in the docs to make it clearer

Reply:
Added to class initializer!